### ISSUE TITLE
Add A Changelog To The Project(s)

### ISSUE THREAD

**Issue by piperRyan** [sozu-proxy/lapin#120]
Hey, I am thinking about leveraging this crate, but right now it is hard to follow where the breaking changes are and what new features are available and so forth.  Accordingly, I think a changelog could help with the individuals following this project to get a quick update on the new versions that are getting published.

**Comment by Keruspe** [comment]
Hi,
I actually wanted to do that before the release but then it slipped out of my mind.
Will do
